app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,19 @@
 <resources>
     <string name="app_name">CompareRxJava1VsRxJava2</string>
+    <string name="buffer">Buffer</string>
+    <string name="flatmap">FlatMap</string>
+    <string name="group_by">GroupBy</string>
+    <string name="map">Map</string>
+    <string name="scan">Scan</string>
+    <string name="window">Window</string>
+    <string name="transforming_operators">Transforming Operators</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.transforming_operators` appears to be unused
 Unused resources make applications larger and slow down builds.